hw/formal/tools/jaspergold/fpv.tcl

```diff
@@ -116,6 +116,15 @@ if {$env(DUT_TOP) == "pinmux_tb"} {
   clock -rate {tl_i, cio_d_i, cio_dp_i, cio_dn_i, cio_sense_i} clk_i
   reset -expr {!rst_ni !rst_aon_ni}
 
+} elseif {$env(DUT_TOP) == "clkmgr"} {
+  clock clk_main_i
+  clock clk_i -both_edges
+  clock clk_io_i -factor 1
+  clock clk_usb_i -factor 1
+  clock clk_aon_i -factor 2
+  clock -rate -default clk_i
+  reset -expr {!rst_ni !rst_main_ni}
```

Comment:
I don't understand what this means... is it missing an operator between the two resets?

Reply:
I believe this is how FPV defines more than one reset pins:
This is from jaspergold support:

A set of pins (input or global stopat net) that are asserted when the design is being reset by the tool and de-asserted when the design is being formally verified.
```
reset -expression only supports pin constraints.
The -expression switch is optional. That is, the following pairs of commands have the same effect:
Example 1:
% reset -expression ~rst1N rst2
% reset ~rst1N rst2
Example 2:
% reset -expression {~rst1N rst2}
% reset {~rst1N rst2}
```

Reply:
Thanks, so it is a list of resets, not a boolean expression.